src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -25,10 +25,6 @@ var fluid_2_0 = fluid_2_0 || {};
             toggleButton: ".flc-slidingPanel-toggleButton",
             toggleButtonLabel: ".flc-slidingPanel-toggleButton"
         },
-        strings: {
-            showText: "show",
-            hideText: "hide"
-        },
```

Comment:
I think you need to keep some default text, for the sliding panel. Actually you might want to add default strings for all of the string options. It seems the sliding panel tests are failing because of this.